publisher/server/publisher.go

```diff
@@ -203,12 +204,24 @@ func max(n []int) int {
 	return val
 }
 
+// getErrorLevel returns the error level
+func (s *Server) getErrorLevel() string {
+	response, err := s.EtcdClient.Get("/deis/publisher/errorLogLevel", false, false)
```

Comment:
@eMxyzptlk what about the overhead of this?
I mean `setEtcd` is invoked from a for loop each 10 seconds so if you have just 10 app is 10 additional `etcd` calls just to get the log level.

Why not check the flag periodically from `etcd` (like 30 or 60 seconds) and "store" the value in a local variable instead of the retrieve the value from `etcd`?